### PR TITLE
UGC-4495 |  Get corret params for infobox

### PR DESCRIPTION
**Pull request by MrVanosh**

### CODE REVIEW

File: includes/Api/ApiTemplateDataHooks.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace MediaWiki\Extension\TemplateData\Api;
+
+/**
+ * This is a hook handler interface, see mediawiki/docs/Hooks.md.
+ */
+interface ApiTemplateDataHooks {
```
Comment (mszabo-wikia) [inline includes/Api/ApiTemplateDataHooks.php]: where is this used?

Comment (MrVanosh) [reply includes/Api/ApiTemplateDataHooks.php]: Here https://github.com/Wikia/unified-platform/pull/15869/files#diff-620fd9b5b921033514a8e2dc7e1bc32d87044cc74ee017f5fa8cbe6e052e1d36

File: modules/ext.templateDataGenerator.data/SourceHandler.js [hunk 2]
```diff
@@ -27,7 +27,9 @@ function SourceHandler( config ) {
 
 	// Config
 	this.setParentPage( config.parentPage );
-	this.setPageSubLevel( config.isPageSubLevel );
+	// Fandom change - begin - UGC-4495 - Make Template be recognized for getting template data
+	this.setPageSubLevel( config.parentPage.includes('Template') ? true : config.isPageSubLevel );
+	// Fandom change - end
```
Comment (mszabo-wikia) [inline modules/ext.templateDataGenerator.data/SourceHandler.js:32]: Q: Were there some upstream changes that require this JS patch?

Comment (MrVanosh) [reply modules/ext.templateDataGenerator.data/SourceHandler.js:32]: I couldn't find one

**Pull request merged**